6-Pipelines/pipelines.ipynb

```diff
@@ -64,7 +64,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
```

Comment:
Just "clear all outputs" for this last lab so that users don't already see all the answers.